agimus_controller/agimus_controller/trajectory.py

```diff
@@ -12,7 +12,7 @@ class TrajectoryPoint:
     robot_configuration: np.ndarray
     robot_velocity: np.ndarray
     robot_acceleration: np.ndarray
-    robot_effort: np.ndarray
+    robot_effort: np.ndarray  # Is robot effort the same as robot torque?
```

Comment:
```suggestion
    robot_effort: np.ndarray
```
This shouldn't be here